doc/final-design.md

```diff
@@ -151,10 +157,19 @@ Get module binary using account address.
 
 # Design architecture
 
+The main parts are:
+- PalletMove _(this repo)_.
```

Comment:
Sometimes it's `pallet-move` and now here we have `PalletMove`.
Let's choose one naming and use it consequently.

Reply:
Done